src/hashcash.cr

```diff
@@ -4,37 +4,52 @@ require "base64"
 require "digest/sha1"
 
 module Hashcash
-  VERSION = "0.1.0"
+  VERSION       = "0.1.0"
+  STAMP_VERSION = 1
```

Comment:
This is better places within the `Hashcash::Stamp` class.

Reply:
To clarify, does this mean STAMP_VERSION should live in Hashcash module or Hashcash::Stamp class?

Reply:
It relates to the stamp version, so keeping it as part of the stamp class seems to make sense.

You may end up wanting to just make it an attribute on stamp objects, then provide [appropriate errors](https://crystal-lang.org/api/0.35.1/NotImplementedError.html) if you are ever passed a non v1 stamp to parse.